pagesections/main.statement.section.js

```diff
@@ -88,7 +88,7 @@ const MainStatementSection = ( Base ) => class extends Base {
 
 		referencesContainer.$(
 			this.constructor.STATEMENT_WIDGET_SELECTORS.EDIT_INPUT_VALUE
-		).waitForExist( null, true );
+		).waitForExist( true );
```

Comment:
I believe the logic that should be followed here is the same as above. Thus I think that this parameter should be `{reverse: true}`